resources/META-INF/plugin.xml

```diff
@@ -31,6 +31,14 @@
       <action id="Flutter.DeviceSelector" class="io.flutter.actions.DeviceSelectorAction"
               description="Flutter Device Selection"
               icon="FlutterIcons.Phone"/>
+      <action id="Flutter.ReloadFlutterAppKey" class="io.flutter.actions.ReloadFlutterAppKeyAction" text="Reload Flutter app">
```

Comment:
I think the text for these actions should be the same as on the debugger toolbar. The names are similar, but do different things - I'd like to make it as easy as possible to distinguish them.

Right now we're using `Full Application Restart` and `Hot Reload`. Perhaps for these keybindings, we use something like `Flutter Restart Application` and `Flutter Hot Reload`?

And in general we should make hot reload easier to get to - slightly more promoted - than the full app restart.